examples/e-commerce/index.html

```diff
@@ -25,8 +25,7 @@
     <link rel="stylesheet" href="./src/app.css" />
     <link rel="stylesheet" href="./src/app.mobile.css" />
     <link rel="stylesheet" href="./src/widgets/PriceSlider.css" />
-    <script src="https://polyfill.io/v3/polyfill.min.js?features=default,Array.prototype.find,Array.prototype.includes"></script>
-
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default%2CArray.prototype.find%2CArray.prototype.includes%2CPromise%2CObject.assign%2CObject.entries"></script>
```

Comment:
IIUC, the new client makes use of new polyfills. 
When it's fully released, it might be worth updating this section for each flavour: https://www.algolia.com/doc/guides/building-search-ui/installation/js/#browser-support

Reply:
Yes, let's not wait for v4 to update that

Reply:
Yes. Noted here: https://github.com/algolia/doc/pull/4053.

Reply:
Is noted here: https://github.com/algolia/doc/pull/4053.